enhancements/update/accepted-risks.md

```diff
@@ -0,0 +1,479 @@
+---
+title: Accepted Risks
```

Comment:
nit: [Markdown linter][1]:

```
enhancements/update/accepted-risks.md: the title Accepted Risks and the file base name accepted-risks must match
```

I think it's asking for `title: accepted-risks`?

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_enhancements/1807/pull-ci-openshift-enhancements-master-markdownlint/1937860667658735616#1:build-log.txt%3A571

Reply:
Done.